docs/deployment/resource-providers/native_kubernetes.md

```diff
@@ -24,394 +23,268 @@ specific language governing permissions and limitations
-<div data-lang="java" markdown="1">
-{% highlight dockerfile %}
-FROM flink
-RUN mkdir -p $FLINK_HOME/usrlib
-COPY /path/of/my-flink-job-*.jar $FLINK_HOME/usrlib/my-flink-job.jar
-{% endhighlight %}
+### Accessing Flink's Web UI
 
-Use the following command to start a Flink application.
-{% highlight bash %}
-$ ./bin/flink run-application -p 8 -t kubernetes-application \
-  -Dkubernetes.cluster-id=<ClusterId> \
-  -Dtaskmanager.memory.process.size=4096m \
-  -Dkubernetes.taskmanager.cpu=2 \
-  -Dtaskmanager.numberOfTaskSlots=4 \
-  -Dkubernetes.container.image=<CustomImageName> \
-  local:///opt/flink/usrlib/my-flink-job.jar
-{% endhighlight %}
-</div>
+Flink's Web UI and REST endpoint can be exposed in several ways via the [kubernetes.rest-service.exposed.type]({% link deployment/config.md %}#kubernetes-rest-service-exposed-type) configuration option.
 
-<div data-lang="python" markdown="1">
-{% highlight dockerfile %}
-FROM flink
+- **ClusterIP**: Exposes the service on a cluster-internal IP.
+  The Service is only reachable within the cluster.
+  If you want to access the Job Manager ui or submit job to the existing session, you need to start a local proxy.
+  You can then use `localhost:8081` to submit a Flink job to the session or view the dashboard.
 
-# install python3 and pip3
-RUN apt-get update -y && \
-    apt-get install -y python3.7 python3-pip python3.7-dev && rm -rf /var/lib/apt/lists/*
-RUN ln -s /usr/bin/python3 /usr/bin/python
+{% highlight bash %}
+$ kubectl port-forward service/<ServiceName> 8081
+{% endhighlight %}
 
-# install Python Flink
-RUN pip3 install apache-flink
-COPY /path/of/python/codes /opt/python_codes
+- **NodePort**: Exposes the service on each Node’s IP at a static port (the `NodePort`).
+  `<NodeIP>:<NodePort>` can be used to contact the Job Manager Service.
+  `NodeIP` can also be replaced with the Kubernetes ApiServer address. 
+  You can find its address in your kube config file.
 
-# if there are third party python dependencies, users can install them when building the image
-COPY /path/to/requirements.txt /opt/requirements.txt
-RUN pip3 install -r requirements.txt
+- **LoadBalancer**: Exposes the service externally using a cloud provider’s load balancer.
+  Since the cloud provider and Kubernetes needs some time to prepare the load balancer, you may get a `NodePort` JobManager Web Interface in the client log.
+  You can use `kubectl get services/<cluster-id>-rest` to get EXTERNAL-IP and then construct the load balancer JobManager Web Interface manually `http://<EXTERNAL-IP>:8081`.
 
-# if the job requires external java dependencies, they should be built into the image as well
-RUN mkdir -p $FLINK_HOME/usrlib
-COPY /path/of/external/jar/dependencies $FLINK_HOME/usrlib/
-{% endhighlight %}
+Please reference the official documentation on [publishing services in Kubernetes](https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types) for more information.
```

Comment:
Here I'm not 100% sure whether `reference` is the right word? 🤔 Alternatively, something like "Please read [...]" might work as well.

Reply:
I'd go with "please refer to the official..."